Transport/SlooceTransport.php

```diff
@@ -90,15 +91,16 @@ public function __construct(
     }
 
     /**
-     * @param string $content
+     * @param string       $content
+     * @param array<mixed> $media
      *
      * @return bool|PluginNotConfiguredException|mixed|string
      *
      * @throws MessageException
      * @throws SloocePluginException
      * @throws \Mautic\IntegrationsBundle\Exception\IntegrationNotFoundException
      */
-    public function sendSms(Lead $contact, $content)
+    public function sendSms(Lead $contact, $content, Stat $stat = null, array $media = [])
```

Comment:
We don't want to break BC. See https://github.com/acquia/mc-cs/pull/1952#discussion_r1135612145 for more info